.jenkins/src/Dcap.groovy

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/groovy
+// Licensed under the MIT License.
```

Comment:
Can you add a comment explaining what this file is for?

Reply:
Sure. I will add comment.